src/services/to-dos.service.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <json/json.h>
+
+struct ToDo {
+    int id;
+    std::string description;
+
+    Json::Value toJson() const;
+};
+
+class ToDos {
```

Comment:
Please name that `ToDoService` if its service, `cos in future we need to create ToDo as model which stores in db